### PR TITLE
Update blocker documentation for changes in RE-338, RE-34, and RE-337

### PR DESCRIPTION
**Pull request by sfbee**
By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

**Review by toddr — APPROVED** [review]

File: docs-website-src/content/blockers.md [hunk 1]
```diff
@@ -106,15 +106,15 @@ We block if your machine has multiple network interface cards (NICs) using kerne
   * The "default" approach in `network-scripts` config files of specifying NICs by `DEVICE` can cause issues due to the above.
   * To find a more in-depth explanation of *why* this is a problem (and what to do about it), read Freedesktop.org's <a href="https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/" target="_blank">Predictable Network Interface Names</a> documentation.
 
-One way to prevent these issues is to assign a custom name in the configuration and re-initialize NICs ahead of time.
+On systems configured in this manner, we will offer to automatically assign a custom names (from ethX to cpethX) in the configuration and re-initialize NICs ahead of elevation.
```
Comment (cPholloway) [inline docs-website-src/content/blockers.md:109]: Saying that we block on it, then saying that we offer to automatically update the NIC name is going to be confusing to anyone reading the documention.  I think we should update this section to something similar to the following:

```
### Multiple network interface cards using kernel names

If your machine has multiple network interface cards (NICs) using kernel names (`ethX`), we offer to automatically update the name from `ethX` to `cpethX` during elevation.
  * To find a more in-depth explanation of *why* this name change is necessary, read Freedesktop.org's <a href="https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/" target="_blank">Predictable Network Interface Names</a> documentation.
```

File: docs-website-src/content/blockers.md [hunk 2]
```diff
@@ -106,15 +106,15 @@ We block if your machine has multiple network interface cards (NICs) using kerne
...
 
 ### MySQL version
 
 A MySQL upgrade **cannot be** in progress.
 
 If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8-based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.
 
-The system **must** not be set up to use a remote database server.
+The system **must** not be set up to use a remote database server. In cases where the system is set up to use a remote server, a local server will be temporarily configured and enabled for the duration of the elevation, and the remote instance will be re-enabled once the elevation completes.
```
Comment (cPholloway) [inline docs-website-src/content/blockers.md:117]: We only do this if the local version of MySQL will be unsupported after the elevation is complete:

```
    # If the database version is supported on the new OS version, then no need to upgrade
    return if Elevate::Database::is_database_version_supported( Elevate::Database::get_local_database_version() );

    $self->_ensure_localhost_mysql_profile_is_active(1);
```

SUGGESTION:

```
### MySQL version

A MySQL upgrade **cannot be** in progress.

If the local version of MySQL/MariaDB installed on the system is not supported on RHEL 8-based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.

**NOTE:** In cases where an upgrade is necessary **and** the system is set up to use a remote server, a local server will be temporarily configured and enabled for the duration of the elevation, and the remote instance will be re-enabled once the elevation completes.
```

**Review by cPholloway — CHANGES_REQUESTED** [review]

File: docs-website-src/content/blockers.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ We do **not** support running the system in a container-like environment.
 
 To run the ELevate script successfully, you will need to be on a version mentioned in the _Latest cPanel & WHM Builds (All Architectures)_ section at http://httpupdate.cpanel.net/.
 
-If you are not on a version mentioned in <a href="http://httpupdate.cpanel.net/" "target=_blank">_Latest cPanel & WHM Builds (All Architectures)_</a> section, run the `/usr/local/cpanel/scripts/upcp` script to update.
+If you are not on a version mentioned in [Latest cPanel & WHM Builds (All Architectures)](http://httpupdate.cpanel.net/) section, run the `/usr/local/cpanel/scripts/upcp` script to update.
```
Comment (cPholloway) [inline docs-website-src/content/blockers.md:65]: Thanks for catching this

**Review by kate-official — APPROVED** [review]
All these changes make sense and look good to me.

**Review by cPholloway — APPROVED** [review]

**Review by WriterAmanda — APPROVED** [review]
The wording looks good 👍

**Pull request merged**